backend/src/routes/auth.ts

```diff
@@ -160,4 +164,114 @@ export const authRoute = new Elysia({ prefix: '/auth' })
             status: 'success',
             message: `${user.email} has been verified`,
         };
+    })
+    .post('/sign-in', async ({ request, cookie, set }) => {
+        const formData = await request.formData();
+        const validEmaillPass = signInFormSchema.safeParse({
+            email: formData.get('email'),
+            password: formData.get('password'),
+        });
+
+        if (!validEmaillPass.success) {
+            return {
+                status: 'error',
+                errors: validEmaillPass.error.flatten().fieldErrors,
+            };
+        }
+        
+        const { email, password } = validEmaillPass.data;
+        
+        let user_id = '';
+        
+        const queryAuthUserData = await sql`
+        SELECT id, hashed_password
+        FROM auth_user
+        WHERE email = ${email}
+        `;
+        
+        const queriedHashedPassword = queryAuthUserData[0].hashed_password;
+        user_id = queryAuthUserData[0].id;
```

Comment:
If user's email isn't valid (not exist in database), will this return undefined instead?.
You should check for `queryAuthUserData.length == 0` then fake hashing to protect from time attack